src/satisfiers/route.ts

```diff
@@ -0,0 +1,22 @@
+import { Satisfier, Caveat } from 'lsat-js'
+import { Request } from 'express'
+
+/**
+ * @description A satisfier for validating caveats based on the original route
+ * used in the exported boltwallConfig ROUTE_CAVEAT_CONFIGS
+ * @type Satisfier
+ */
+const routeSatisfier: Satisfier = {
+  condition: 'route',
+  satisfyFinal: (caveat: Caveat, req: Request) => {
+    const master = req.boltwallConfig?.masterRoute
+    let pathMatches
+    if(req.boltwallConfig?.allowSubroutes) {
+      pathMatches = (req.path+'/').startsWith(caveat.value+'/')
+    } else pathMatches = (caveat.value === req.path)
+    if (caveat.condition === 'route' && (pathMatches || caveat.value === master)) return true
```

Comment:
Yes! I thought this was a interesting idea. Could be a useful way to provide all access pass to your endpoints.

I've also thought about how it would be cool to be able to combine a few of these configs together. Maybe you want both time caveat and route caveat. I'd have to think about it a bit more how to impliment. Maybe that could be another PR in the future.

Reply:
Yeah totally. The config object itself does allow for composability which you can see in the [type definition](https://github.com/Tierion/boltwall/blob/master/src/typings/configs.d.ts#L31-L39) where `getCaveats` and `caveatSatisfiers` can be arrays in which case each one would be added in sequence. So it would be nice to be able to pass in an array of configs you want to compose. There's some trickiness around the other config values which is probably why I didn't add that originally (e.g. invoice description, amounts, rate) but that could just be overridden as necessary.